tests/accessor/local_accessor_access_among_work_items.h

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests that the sycl::local_accessor can access the memory shared
+//  among work-items.
+//
+*******************************************************************************/
+#ifndef SYCL_CTS_LOCAL_ACCESSOR_ACCESS_AMONG_WORK_ITEMS_H
+#define SYCL_CTS_LOCAL_ACCESSOR_ACCESS_AMONG_WORK_ITEMS_H
+#include "accessor_common.h"
+
+namespace local_accessor_access_among_work_items {
+using namespace sycl_cts;
+using namespace accessor_tests_common;
+
+/**
+ * @brief Provides a functor that provides verification that local_accessor can
+ *        access the memory shared among work-items
+ * @tparam T Current data type
+ * @tparam DimensionTypeT Current current dimension size
+ */
+template <typename T, typename DimensionTypeT>
+class run_test {
+  static constexpr int Dimension = DimensionT::value;
+
+ public:
+  /**
+   * @brief Functor that provides verification that local_accessor can access
+   *        the memory shared among work-items
+   * @param type_name Current data type string representation
+   */
+  void operator()(const std::string& type_name) {
+    auto queue = util::get_cts_object::queue();
+
+    auto section_name = get_section_name<Dimension>(
+        type_name,
+        "Verify possibility to access the memory shared among work-items. "
+        "[local_accessor]");
+    SECTION(section_name) {
+      T values_arr[2] = {1, 2};
+      const sycl::range range(2);
+      constexpr T new_value = 5;
+      bool is_acc_val_equal_to_expected = false;
+      {
+        sycl::buffer<bool> val_is_equal_to_expected_buffer(
+            &is_acc_val_equal_to_expected, sycl::range(1));
+        queue.submit([&](sycl::handler& cgh) {
+          auto val_is_equal_to_expected_acc =
+              val_is_equal_to_expected_buffer
+                  .template get_access<sycl::access_mode::write>(cgh);
+          sycl::local_accessor<T, Dimension> acc(range, cgh);
+          cgh.parallel_for(
+              sycl::nd_range(range, range), [=](sycl::nd_item item) {
+                T& acc_elem = acc[0];
+
+                // We going to set value in one work-item, synchronize
+                // work-items by sycl::group_barrier() calling and check that
+                // value have been set correctly from another work item.
+                if (item.get_local_id(0) != 0) {
+                  acc_elem = new_value;
+                }
```

Comment:
Here must be an else-branch which sets "invalid" value. If acc memory is not shared, the part of the memory accessed by work-item 0 is left uninitialized and behavior is undefined.

Reply:
If I correctly understood you.
`is_acc_val_equal_to_expected` has `false` as default value, if `accessor` memory isn't shared, then `is_acc_val_equal_to_expected` will not been changed in `true`, so test will fail

Reply:
No, I'm talking about local memory accessed via `acc`.

Reply:
I've aplied this comment in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/364/commits/1b06b8c73ad78493043c4fcb9804392e6739e941